modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/CreationMenu.js

```diff
@@ -23,6 +23,7 @@ const CreationMenu = ({
 	maxSecondaryItems,
 	maxTotalItems = 15,
 	onCreateButtonClick,
+	onCreationMenuItemClick,
```

Comment:
It's deja vu all over again :slightly_smiling_face: 

We're sending this already in https://github.com/drewbrokke/liferay-portal/pull/1301.

It's the same commit, so :+1:  Just noting this for anybody else reviewing this PR.

Reply:
:+1: thanks

Reply:
Yeah the thing is that PR hasn't been merged yet, so I just cherry-picked.